test/topology_custom/test_commitlog.py

```diff
@@ -0,0 +1,92 @@
+#
+# Copyright (C) 2022-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+
+import asyncio
+import pytest
+import logging
+import time
+from test.pylib.manager_client import ManagerClient
+from test.pylib.random_tables import RandomTables
+from test.pylib.util import wait_for_cql_and_get_hosts
+from test.topology.util import reconnect_driver
+from test.topology.conftest import skip_mode
+from test.pylib.random_tables import Column, TextType
+
+logger = logging.getLogger(__name__)
+
+@pytest.mark.asyncio
+@skip_mode('release', 'error injections are not supported in release mode')
+async def test_reboot(request, manager: ManagerClient):
+    # Check that commitlog provides durability in case of a node reboot.
+    #
+    # 1. start a node with commitlog_sync: batch
+    # 2. create a test table t
+    # 3. write some data to the table and truncate it, now a truncation record should be created
+    # 4. cleanly restart a node with enabled injection decrease_schema_commitlog_base_segment_id,
+    #    this allows to set new segment id to zero, which imitates node reboot
```

Comment:
I don't understand this, you cleanly restart a node, so what's the deal with "imitating node reboot" when in fact you just did a node reboot?